dist/images/ovndb-raft-functions.sh

```diff
@@ -238,26 +274,53 @@ ovsdb-raft() {
     ovn_raft_conn_ip_url_suffix=":[::]"
   fi
 
-  if [[ "${POD_NAME}" == "ovnkube-db-0" ]]; then
-    run_as_ovs_user_if_needed \
-      ${OVNCTL_PATH} run_${db}_ovsdb --no-monitor \
-      --db-${db}-cluster-local-addr=[${ovn_db_host}] \
-      --db-${db}-cluster-local-port=${raft_port} \
-      --db-${db}-cluster-local-proto=${transport} \
-      ${db_ssl_opts} \
-      --ovn-${db}-log="${ovn_loglevel_db}" &
-  else
-    # join the remote cluster node if the DB is not created
-    if [[ "${initialize}" == "true" ]]; then
-      wait_for_event ready_to_join_cluster ${db} ${port}
-    fi
-    run_as_ovs_user_if_needed \
+  initial_raft_create=true
+  if [[ "${initialize}" == "true" ]]; then
```

Comment:
ovsdb-server needs to be started when ovnkube-db-raft pod restarts (where $initialize is false).

Reply:
thanks. will fix.